eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/Serialisation/Generators.hs

```diff
@@ -227,8 +229,8 @@ genCM lang costModelParamNames = do
   either (error "Corrupt cost model") pure $ mkCostModel lang newCMPs
 
 genCostModel :: Language -> Gen (Language, CostModel)
-genCostModel PlutusV1 = (PlutusV1,) <$> genCM PlutusV1 PV1.costModelParamNames
-genCostModel PlutusV2 = (PlutusV2,) <$> genCM PlutusV2 PV2.costModelParamNames
+genCostModel PlutusV1 = (PlutusV1,) <$> genCM PlutusV1 (Set.fromList (pack . showParamName <$> enumerate @PV1.ParamName))
+genCostModel PlutusV2 = (PlutusV2,) <$> genCM PlutusV2 (Set.fromList (pack . showParamName <$> enumerate @PV2.ParamName))
```

Comment:
With my suggestion of `costParamsNamesSet`  this function becomes:
```suggestion
genCostModel lang = (,) lang <$> genCM lang (costParamsNamesSet lang)
```

Note that `TupleSections` extension can also be removed

Reply:
Addressed this one too